plugins/woocommerce/includes/admin/meta-boxes/views/html-product-attribute.php

```diff
@@ -11,6 +11,88 @@
 		<strong class="attribute_name<?php echo esc_attr( $attribute->get_name() === '' ? ' placeholder' : '' ); ?>"><?php echo esc_html( $attribute->get_name() !== '' ? wc_attribute_label( $attribute->get_name() ) : __( 'Custom attribute', 'woocommerce' ) ); ?></strong>
 	</h3>
 	<div class="woocommerce_attribute_data wc-metabox-content hidden">
-		<?php require __DIR__ . '/html-product-attribute-inner.php'; ?>
+		<table cellpadding="0" cellspacing="0">
+			<tbody>
+				<tr>
+					<td class="attribute_name">
+						<label><?php esc_html_e( 'Name', 'woocommerce' ); ?>:</label>
+
+						<?php if ( $attribute->is_taxonomy() ) : ?>
+							<strong><?php echo wc_attribute_label( $attribute->get_name() ); ?></strong>
```

Comment:
We should escape this, like we did in the `html-product-attribute-inner.php` file...

```php
esc_html( wc_attribute_label( $attribute->get_name() ) )
```